src/Combinatorics/EnumerativeCombinatorics/combinations.jl

```diff
@@ -18,10 +18,10 @@ julia> collect(C)
  [2, 3, 4]
 ```
 """
-combinations(n::Int, k::Int) = Combinations(Base.OneTo(n), k)
+combinations(n::IntegerUnion, k::IntegerUnion) = Combinations(Base.OneTo(n), k)
```

Comment:
ahhh I see, that's how it worked in the example above :-)

I think we need to make clear it in the docstring which of the two types control the output type (only the first one, as far as I can tell).

In that sense I kinda prefer the version where we only allow `Int` for `k` (less confusion). But perhaps for a user it's more convenient to be allowed to pass in any kind of integer for `k`... *shrug*